README.md

```diff
@@ -11,6 +11,7 @@ A Typescript Action that performs HTTP requests within your workflow. It support
 | url | Endpoint to query | https://api.github.com/graphql |
 | method | HTTP method | `GET` |
 | data | HTTP request payload |  |
+| headers | HTTP headers | {'Content-Type': 'application/json'} |
```

Comment:
This is technically not true for all inputs 😅the default values are defined by by the [`actions.yml`](https://github.com/CamiloGarciaLaRotta/watermelon-http-client/pull/25/files#diff-689e1a63f106e616debbed3b5aedcb43R15-R16). But you are correct with respect to GraphQL.
```suggestion
| headers | HTTP headers | Empty by default. If no headers are supplied, GraphQL queries will be sent with `{'Content-Type': 'application/json'}` |
```

Reply:
Changed it to : DummyToken123

Reply:
ooh that's true. My bad. Fixed.